package/kernel/mwlwifi/Makefile

```diff
@@ -8,16 +8,16 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mwlwifi
-PKG_RELEASE=2
+PKG_RELEASE=1
```

Comment:
Should not change, this is first release of version bump

Reply:
Shouldn't this be reset to 1 since it's a new driver version?